Replicator/RevFinder.cc

```diff
@@ -281,9 +281,15 @@ namespace litecore::repl {
                     logDebug("    - Already have '%.*s' %.*s but need to mark it as remote ancestor",
                              SPLAT(docID), SPLAT(revID));
                     _db->setDocRemoteAncestor(docID, revID);
-                    if (!_passive && !_db->usingVersionVectors())
-                        replicator()->docRemoteAncestorChanged(alloc_slice(docID),
-                                                               alloc_slice(revID));
+                    if (!_passive && !_db->usingVersionVectors()) {
+                        auto repl = replicatorIfAny();
+                        if(repl) {
+                            replicator()->docRemoteAncestorChanged(alloc_slice(docID),
```

Comment:
Shouldn't this call `repl` instead of `replicator()`?

Reply:
yes. limitation of manual porting. Thanks a lot. Otherwise it is going to be a random bomb.